docs/utils/mdHelper.js

```diff
@@ -0,0 +1,29 @@
+// @flow strict
+import path from 'path';
+import fs from 'fs';
+import matter from 'gray-matter';
+
+export function getDocByRoute(
+  route: string,
+): {| content?: string, meta: { [key: string]: string }, route: string, isMDX: boolean |} {
+  const docsDirectory = path.join(process.cwd(), 'markdown');
+  const fullPath = path.join(docsDirectory, `${route}.md`);
+
+  try {
+    const fileContents = fs.readFileSync(fullPath, 'utf8');
+    const { data, content } = matter(fileContents);
+
+    return { route, meta: data, content, isMDX: true };
+  } catch (ex) {
```

Comment:
we don't need the ex no?


`  } catch () {`

Reply:
Good call out. 

A few things might go wrong here,

1. The file doesn't exist
2. The frontmatter parsing fails

Tbh, not sure how to handle showing the error. We return a 404, if the page happens to fail, and mark `isMDX` as `false`. Since these are server side rendered, I think if this were to fail, we'd catch it before it ships, since it's not client dependent, and perhaps that's okay.

Reply:
Perhaps we can use a Google Analytics event to log the error?

Reply:
Added GA Logging, not sure the naming convention is right, but I didn't notice any precedent outside of button clicks atm